compiler/rustc_borrowck/src/diagnostics/mutability_errors.rs

```diff
@@ -183,9 +201,14 @@ impl<'a, 'tcx> MirBorrowckCtxt<'a, 'tcx> {
         let mut mut_error = None;
         let mut count = 1;
 
+        let cant_access_here = |sp: Span| match error_access {
+            AccessKind::MutableBorrow => FnMutBumpFn::CannotBorrowMut { sp },
+            AccessKind::Mutate => FnMutBumpFn::CannotAssign { sp },
+        };
         let span = match error_access {
+            // cannot assign only happen in Mutate
```

Comment:
This comment is a quite confusing, it could easily be describing the behaviour of the compiler, rather than the ability to assign to `diagnosing`.

Reply:
It was a note for myself to locate the `cannot_assign()` for debugging a `eager_subdiag` case, I should have removed it.